### ISSUE TITLE
Draw focus ring on Omnibar

### ISSUE THREAD

**Issue by willco007** [harveycabaguio/firefox-elementary-theme#18]
When the omnibar has focus, the text changes from gray to black, but it would be nice to have the blue focus ring like Epiphany does to further indicate it has focus.
